lib/config/aws.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import (
+	"github.com/apex/log"
+	"github.com/aws/aws-sdk-go/aws"
+)
+
+func (this *config) GetAws() *aws.Config {
+	if this.Aws == nil {
+		log.Debug("AWS Config not defined. Use default config.")
+		this.Aws = &aws.Config{Region: aws.String("us-east-1")}
```

Comment:
Good catch, I've added a command line arg for kinesis. Adding this to the s3 section would take a bit more work, but it's available as the AWS_REGION environment variable as well for future compatibility.